src/test/java/api/loans/DueDateScheduledNoticesProcessingTests.java

```diff
@@ -384,11 +403,28 @@ void testNoticeIsNotSentOrDeletedWhenPatronNoticeRequestFails() {
 
     checkSentNotices();
 
-    verifyNumberOfScheduledNotices(1);
+    verifyNumberOfScheduledNotices(0);
     verifyNumberOfPublishedEvents(NOTICE, 0);
     verifyNumberOfPublishedEvents(NOTICE_ERROR, 1);
   }
 
+  @Test
+  void testErrorEventsAreSentWhenUpdateNoticeRequestFails() {
+    generateLoanAndScheduledNotices();
+
+    createNoticesOverTime(dueDate.minusMinutes(1)::minusHours, 2).forEach(
+      scheduledNoticesClient::create);
+
+    FakeStorageModule.setFailDeleteWithBadRequest(true);
+    FakeStorageModule.setFailUpdateWithBadRequest(true);
```

Comment:
Make flags non-static

Reply:
Deleted, as we cannot access fake storage instances